apps/frontend/src/libs/helpers/get-formatted-date.ts

```diff
@@ -0,0 +1,12 @@
+import { formatInTimeZone } from "date-fns-tz";
+
+type FormatDate = "dd MMM yyyy" | "yyyy MM dd";
+
+const getFormattedDate = (
+	date: Date | string,
+	formatDate: FormatDate,
+): string => {
+	return formatInTimeZone(new Date(date), "UTC", formatDate);
```

Comment:
let's convert it in a place where we call it, to reduce possible options
```suggestion
	date: Date,
	formatDate: FormatDate,
): string => {
	return formatInTimeZone(data, "UTC", formatDate);
```